CIP-ledger-evolution/Serialization.md

```diff
@@ -0,0 +1,97 @@
+# Serialization in the ledger
```

Comment:
I still don't really understand why the ledger has to specify the serialisation format of data structures: It could perfectly well be defined by leaving these abstract. I understand some rules of the ledger require that some signature/hash be valid which implies there's some binary format somewhere but why make it part of the ledger?

Reply:
The wire format for transactions and blocks is very important to a wide group of people, and these are definitely ledger topics. I think it is necessary to have the serialization change discussions clarified and open. I authored this CIP in November, https://github.com/cardano-foundation/CIPs/pull/372, and thought that it's proper category would be "ledger".

How would you handle the process around changes to the transaction and block serializations?